pettingzoo/butterfly/knights_archers_zombies/knights_archers_zombies.py

```diff
@@ -50,6 +50,8 @@ def __init__(
         pad_observation=True,
         line_death=False,
         max_cycles=900,
+        vector_state=None,
```

Comment:
Maybe this should be True/False?

Reply:
I've fixed this according to your suggestion